README.md

```diff
@@ -27,6 +28,44 @@ ARGUMENTS
       input-file (required)   The CSV file (RFC4180) with required changes. The first line must be
                               'FEDORA_ID,STREAM_ID,XML_TAG,OLD_VALUE,NEW_VALUE', in that order. Additional columns
                               and empty lines are ignored.
+      additional-directory (optional)
+                              A reference to a directory containing additional files
+                              
+EXAMPLES
+--------
+
+**Example: update simple value in EMD**
+
+This module was developed to update simple values in the EMD, like the AccessCategory. 
+```$csv
```

Comment:
I think the dollar sign [should not be there](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).